app/models/document.rb

```diff
@@ -75,6 +86,18 @@ def published?
     self.class.for_publication.where(id: id).any?
   end
 
+  def created_by
```

Comment:
this is presentational logic, it shouldn't live here - `document_helper.rb` feels like the right place